code/modules/mob/living/living.dm

```diff
@@ -756,11 +756,15 @@
 	return "You can't fit into that vent."
 
 //called when the mob receives a bright flash
-/mob/living/proc/flash_eyes(intensity = 1, override_blindness_check = 0, affect_silicon = 0, visual = 0, type = /obj/screen/fullscreen/flash)
+/mob/living/proc/flash_eyes(intensity, override_blindness_check, affect_silicon, visual, type)
```

Comment:
The default values here are gone. Meaning all calls depending on those will break

Reply:
Yeah, this needs to be addressed.